composer.json

```
@@ -34,7 +34,7 @@
         "phpunit/phpcov": "^8.2",
         "phpunit/phpunit": "^9.1",
         "predis/predis": "^1.1 || ^2.0",
-        "rector/rector": "0.19.8",
+        "rector/rector": "1.0.0",
```

@samsonasik Is this okay?
```suggestion
        "rector/rector": "^1.0",
```

---

I still prefer pinned version for rector for CodeIgniter repo, that will easy to verify regression on each version tagged, as rector rely on phpstan, and on some condition, the tweak or version up requirement in rector can be needed

---

Okay, then let's go with pinned version in this repository.